app/src/main/java/org/mozilla/focus/theme/FocusTheme.kt

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.focus.theme
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.material.Colors
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.darkColors
+import androidx.compose.runtime.Composable
+
+/**
+ * The theme used for Firefox Focus/Klar for Android.
+ */
+@Composable
+fun FocusTheme(
+    darkTheme: Boolean = isSystemInDarkTheme(),
+    content: @Composable () -> Unit
+) {
+    MaterialTheme(
+        colors = if (darkTheme) darkColorPalette else lightColorPalette,
+        content = content
+    )
+}
+
+private val darkColorPalette: Colors = darkColors()
```

Comment:
I think this will add the Material default colors to FocusTheme, not the ones we use. We could add our own colors we already use with something in the lines of :` darkColors (primary = colorResource(id = R.color.primary) )` etc...

Reply:
I'll update to the latest A-C version here, which should fix the text color problem. I'll also add the background color here.